charts/latest-v2/azuredisk-csi-driver/values.yaml

```diff
@@ -58,6 +58,7 @@ controller:
   cloudConfigSecretName: azure-cloud-provider
   cloudConfigSecretNamespace: kube-system
   allowEmptyCloudConfig: false
+  enableMountReplicas: false
```

Comment:
The value for this in our latest-v2 chart should be `true` I believe.

Reply:
In the PRD, the default value is set to false.

Reply:
fixed